### PR TITLE
Add flags and tooltip

### PR DESCRIPTION
**Pull request by meyskens**
schermafbeelding 2016-08-26 om 15 42 34

---
This change is [Reviewable](https://reviewable.io/reviews/innovate-technologies/control/59)

### CODE REVIEW

File: src/manage/cast/statistics/index.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
 import "./statistics.css";
+import "flag-icon-css/css/flag-icon.css";
+import "angular-tooltips";
+import "angular-tooltips/dist/angular-tooltips.min.css";
```
Comment (leoetlino) [inline src/manage/cast/statistics/index.js]: Can you use the non dist version instead? Webpack handles the whole build process (minification, etc.) by itself, and I remember it works best on unminified files and also makes development easier (by seeing the original CSS).

File: package.json [hunk 2]
```diff
@@ -25,11 +25,13 @@
     "angular-sanitize": "1.4.8",
     "angular-smart-table": "^2.1.6",
     "angular-strap": "^2.3.6",
+    "angular-tooltips": "^1.1.6",
```
Comment (leoetlino) [inline package.json]: We already have a way to show tooltip, IIRC.

File: src/manage/cast/statistics/index.js (hunk 1 above)
Comment (meyskens) [inline src/manage/cast/statistics/index.js]: It uses scss for build, will see if there is an on unminified one.

File: package.json (hunk 2 above)
Comment (meyskens) [inline package.json]: Do we?

Comment (leoetlino) [inline package.json]: Yes, our Bootstrap UI library (ng-strap) has support for tooltips. And I think we already use them in some places.
just grep for `tooltip` in src/

File: src/manage/cast/statistics/statistics.html [hunk 3]
```diff
@@ -16,13 +16,15 @@ <h4 class="list-group-item-heading">{{stream.stream}}</h4>
                     <table class="table table-striped table-hover">
                         <thead>
                             <tr>
+                                <th></th>
```
Comment (leoetlino) [inline src/manage/cast/statistics/statistics.html]: Maybe make this the same column as the IP column? (colspan)

**Comment by leoetlino** [comment]
:lgtm:, this just needs a squash.

---

Reviewed 3 of 3 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/innovate-technologies/control/59#-:-KQC_-kyTETwec0_u-Pe:bo284xi)*

**Pull request merged**